torchmdnet/calculators.py

```diff
@@ -1,9 +1,33 @@
 import torch
 from torchmdnet.models.model import load_model
 
+# dict of preset transforms
+tranforms = {
+    "eV/A": lambda energy, forces: (
+        energy * 23.0609,
```

Comment:
These preset transforms seem to assume the original units are eV and A.  But that isn't generally true.  TorchMD makes no assumptions about units.  The forces and energies are in whatever units the training data happened to be in.

Reply:
No assumption really. The idea is to take into account some possibility directly in the converter without the necessity to write the transform function by yourself. So I put eV/A, Hartree/A and Hartree/Bohr. If you have other units of measure, you have just to specify your function as `output_transform` argument in your yaml file to run torchmd.

Reply:
You do have an assumption: that if they say "eV/A", that means to multiply by the value that would convert eV/A to kcal/mol/A.  Why kcal/mol/A?  That's an arbitrary choice.  You're picking one set of units as preferred over all others, which we haven't done till now and shouldn't do.

Reply:
This because torchmd works with kcal/mol/A.

Reply:
Maybe rename the transform to "eV/A -> kcal/mol/A"

Reply:
@peastman torchmd uses kcal/mol and A, torchmd-net is unit anostic. The proposed calculator is the interface between the simulation code torchmd and NNPs from torchmd-net. Therefore the transformation from any unit (torchmd-net) to kcal/mol and A (torchmd).

@stefdoerr I think the name "eV/A" makes sense as it indicates the source units in the conversion. The target unit is always the same so calling the transform "kcal/mol/A" contains no information.

Reply:
Why so? It contains information to what units you are converting. It's not clear otherwise you are converting them specifically for torchmd

Reply:
@stefdoerr I misunderstood your message which I only realized after my response, I edited my message but I guess the notification you got didn't include that.

Reply:
There's no mention of TorchMD anywhere in the class documentation.  And if this really is specific to it, shouldn't it go in the TorchMD repo instead of this one?

Also, expecting people to pick up on the distinction between TorchMD and TorchMD-Net when looking at this class is perhaps overly subtle! :)

Reply:
I think it is fine here as TorchMD just treats this as an "external potential", which theoretically could be anything.

I think with @stefdoerr's suggestion to rename the presets to follow the style of "eV/A -> kcal/mol/A" everything should be clear?

Reply:
I think it is fine here as TorchMD just treats this as an "external potential", which theoretically could be anything.

I think with @stefdoerr's suggestion to rename the presets to follow the style of "eV/A -> kcal/mol/A" everything should be clear?

Reply:
Sounds good to me.

Reply:
Perfect, I will modify the variable to call the preset transformer as Stefan suggested